GitHub-trainings/github-pr-training/latex-slides/GitHub-PR-training.tex

```diff
@@ -0,0 +1,584 @@
+	\begin{columns}[c]
+		
+		\column{.6\textwidth} % Right column and width
+		\vspace{-.5cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/create-pr-2.png}
+		\end{figure}
+		
+		\column{.4\textwidth} % Left column and width
+		
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item A draft PR has the color code gray 
+			\item You can follow someone's work here - but no need to do anything yet
+			\item The author will eventually remove the draft status and ask for reviews
+		\end{itemize}
+		
+	\end{columns}	
+\end{frame}
+
+\begin{frame}
+	\frametitle{Stage: The work stage}
+	
+	\huge\centering \textbf{WORK STAGE}
+	
+\end{frame}
+
+\begin{frame}
+	\frametitle{Stage: The work stage}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item Most time will be spent in this stage
+			\item Reviewing the code should be seen as important as writing it
+			\item This stage will be repeated until the team agrees on the quality of the code
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S2.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Make commits}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item The author commits edits as usual
+			\item The draft PR will be updated as more commits are pushed to the branch		
+			\item Keep pushing commits until the first version implementation of the task is completed
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S2-1.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Update the PR}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item Remove the draft status from the PR
+			\item Document what has been implemented in the PR
+			\item Assign one or several people to review your PR
+			\item Notify the assignee(s) using email or slack as GitHub notifications may be muted
+		\end{itemize}
+		
+		\column{.65\textwidth} % Right column and width
+		\vspace{-.75cm}
+		\begin{figure}
+			\centering
+			\includegraphics[width=\textwidth]{./img/branch-pr-merge-cycle-S2-2.png}
+		\end{figure}
+		
+	\end{columns}
+\end{frame}
+
+\begin{frame}
+	\frametitle{Update the PR}
+	\begin{columns}[c]
+		
+		\column{.35\textwidth} % Left column and width
+		\begin{itemize}
+			\setlength\itemsep{1em}
+			\item This step is incredible important for high-quality code
+			\item Use the PR to review code edits in the \textit{file changes} tab
+			\item The reviewer adds code suggestions and comments
+		\end{itemize}
+		
```

Comment:
Show how to make comments and suggestions?

Reply:
Done in daa460027cd223043d9667e739327be24d15c4fb